openwisp_users/api/urls.py

```diff
@@ -9,6 +10,24 @@ def get_api_urls(api_views=None):
     urlpatterns = []
     if api_views is None:
         api_views = views
+    urlpatterns += [
+        path('users/organization/', views.organization_list, name='organization_list',),
+        path(
+            'users/organization/<str:pk>/',
+            views.organization_detail,
+            name='organization_detail',
+        ),
+        path('users/user/', views.user_list, name='user_list',),
+        path('users/user/<str:pk>/', views.user_detail, name='user_detail'),
+        path(
```

Comment:
When I submit form without any information, it returns that I have entered wrong information.
This might be confusing when the consumer of the API has a typo or something, we should return, "please enter old and new  passwords".

Reply:
Fixed, :+1: